pkgs/tools/misc/sacad/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, python3, enableCompression ? true, jpegoptim, optipng }:
+
+assert enableCompression -> jpegoptim != null && optipng != null;
+
+with python3.pkgs;
+let
+  inherit (python3.pkgs) buildPythonApplication fetchPypi;
+in
+buildPythonApplication rec {
+  pname = "sacad";
+  version = "2.0.6";
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.